.github/workflows/release/Dockerfile.windows

```diff
@@ -0,0 +1,37 @@
+ARG GO_VERSION=1.16.4
+
+FROM --platform=$BUILDPLATFORM golang:${GO_VERSION} AS base
+RUN apt-get update && apt-get install -y \
+	binutils-mingw-w64 \
+	g++-mingw-w64-x86-64
+ARG TARGETARCH
+ENV CGO_ENABLED=1
+ENV GOOS=windows
+ENV GOARCH=$TARGETARCH
+SHELL ["/bin/bash", "-xec"]
+
+FROM base as base-amd64
+ENV CC=x86_64-w64-mingw32-gcc
+
+FROM base-${TARGETARCH}${TARGETVARIANT} AS base-target
+
+FROM base-target AS containerd
+COPY . /go/src/github.com/containerd/containerd
+WORKDIR /go/src/github.com/containerd/containerd
+RUN make binaries; rm bin/containerd-stress.exe
+
+FROM base-target AS runhcs
+WORKDIR /go/src/github.com/microsoft/hcsshim
```

Comment:
Should probably capitalize `Microsoft` where it's used as part of `GOPATH`, since that's how it's usually imported for hcsshim.